### PR TITLE
Add simple content for each project.

### PR DESCRIPTION
**Pull request by 197g**
Simplistic but filled project pages, short summaries and title cards
should at least get the point across that the organization is not an
effort for a small number of crates but should span multiple, stronger
together than alone.

### CODE REVIEW

**Comment by birktj** [comment]
This will be a nice improvement. Maybe you could add some more text to the image project as well, right now it doesn't really say anything?

**Comment by 197g** [comment]
Of course I forgot the most important one while adding text for the new files :smile:

File: _summaries/image-tiff.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: "tiff"
+---
+
+Decoder for the `tiff` file format.
```
Comment (birktj) [inline _summaries/image-tiff.md]: Remember that this now is both an encoder and decoder.

File: _projects/image.md [hunk 2]
```diff
@@ -9,4 +9,16 @@ docs: "https://docs.rs/image"
...
+more popular encoders (such as `png`, `gif`, `tiff`) where split into separate
+crates. In 2019 the whole collection of image libraries was split into its own
+organization.
+
+The fractal in background of the header of this page was created with the image
+library! It is a variation of a long-standing
+[example](https://github.com/image-rs/image/blob/master/examples/fractal.rs)
+that can be found in the repository. The best part is that the crate is popular
+enough that you can expirement directly on [the Rust Playground
+play.rust-lang.org](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2e8024c1a0fdcb9aebfab712e6f0a32d).
```
Comment (birktj) [inline _projects/image.md:25]: This is nice and all, but afaik you can't really get any image output on the rust playground, so it's maybe not that interesting?

Comment (197g) [reply _projects/image.md:25]: True, it should really be a `wasm` in-browser playground that allows one to 'download' output images. But maybe trying out usage/syntax is already some incentive?

File: _summaries/image-tiff.md (hunk 1 above)
Comment (197g) [reply _summaries/image-tiff.md]: Right, I'll amend it.

File: _projects/image.md (hunk 2 above)
Comment (birktj) [reply _projects/image.md:25]: Yeah, just knowing that the crate can be used on the playground and for checking out the api is perfectly good enough reason to include the link.

Comment (birktj) [reply _projects/image.md:25]: Maybe we can push the playground people towards implementing some sort of support for either downloading files or even better, viewing generated images directly in the playground?

File: _summaries/image-gif.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+name: "tiff"
```
Comment (birktj) [inline _summaries/image-gif.md]: I guess you mean `gif`?

Comment (197g) [reply _summaries/image-gif.md]: Turns out I forgot to adjust the metadata all around.

File: _summaries/image-png.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+name: "png"
+---
+
+Decoder and Encoder for the wide spread `png` format.
```
Comment (birktj) [inline _summaries/image-png.md]: Sorry about the nitpicking, but it is nice to fix stuff like this. Shouldn't `encoder` we be written with a small "e" here, and I think widespread is one word.

File: _projects/image-tiff.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+layout: project
+name: "tiff"
+title: "Tiff"
+weight: 700
+crates_io: "https://crates.io/crates/tiff"
+crates_io_badge: "https://img.shields.io/crates/v/tiff.svg"
+github: "https://github.com/image-rs/image-tiff"
+docs: "https://docs.rs/tiff"
+docs_badge: "https://docs.rs/tiff/badge.svg"
+title: "Tiff"
```
Comment (birktj) [inline _projects/image-tiff.md]: Probably doesn't matter, but `title` appeared twice here. And also it should probably say "decoder and encoder" just bellow here as well.

File: _summaries/image-png.md (hunk 4 above)
Comment (197g) [reply _summaries/image-png.md]: All good, fixing all the nits.

**Review by birktj — APPROVED** [review]
Looks great now! This will definitely improve the website quite a bit.

**Pull request merged**